LICENSE

```diff
@@ -1,4 +1,4 @@
-Copyright (c) 2014 Jamie Gaskins
+Copyright (c) 2014, 2015 Jamie Gaskins
```

Comment:
Straight up, I'm not even sure that's what you're supposed to do, I just saw other people do it :P